File: RuntimeData/runtime/app/src/main/java/Movements/CMoveDefMouse.java
package Movements;

import Services.CFile;

public class CMoveDefMouse extends CMoveDef
{
    public int mmDx;      				
    public int mmFx;
    public int mmDy;
    public int mmFy;
    public int mmFlags;

	public CMoveDefMouse()
	{
	}
	
	@Override
    public void load(CFile file, int length)
    {
        mmDx=file.readAShort();
        mmFx=file.readAShort();
        mmDy=file.readAShort();
        mmFy=file.readAShort();
        mmFlags=file.readAShort();
    }
}

File: RuntimeData/xnaruntime/RuntimeXNA/RuntimeXNA/Movements/CMoveDefMouse.cs
//----------------------------------------------------------------------------------
//
// CMOVEDEFMOUSE : données du mouvement mouse
//
//----------------------------------------------------------------------------------
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using RuntimeXNA.Services;
 
namespace RuntimeXNA.Movements
{
    class CMoveDefMouse : CMoveDef
    {
        public short mmDx;
        public short mmFx;
        public short mmDy;
        public short mmFy;
        public short mmFlags;

        public override void load(CFile file, int length)
        {
            mmDx=file.readAShort();
            mmFx=file.readAShort();
            mmDy=file.readAShort();
            mmFy=file.readAShort();
            mmFlags=file.readAShort();
        }
    }
}
